platforms/windows/hax_wrapper.c

```diff
@@ -237,47 +226,70 @@ void hax_disable_irq(void)
     asm_disable_irq();
 }
 
-void hax_error(char *fmt, ...)
+int hax_log_level(int level, const char *fmt,  ...)
 {
     va_list arglist;
+    va_start(arglist, fmt);
 
+    if (level >= HAX_LOG_DEFAULT)
+        vDbgPrintExWithPrefix("haxm: ", DPFLTR_IHVDRIVER_ID, DPFLTR_ERROR_LEVEL, fmt, arglist);
+
+    va_end(arglist);
+    return 0;
+}
+
+void hax_error(char *fmt, ...)
+{
+    va_list arglist;
     va_start(arglist, fmt);
-    if (HAX_LOGE >= default_hax_log_level)
-        vDbgPrintExWithPrefix("haxm_error:", -1, 0, fmt, arglist);
+
+    if (HAX_LOGE >= HAX_LOG_DEFAULT)
```

Comment:
hax_log_level(HAX_LOGE, fmt, arglist);

Reply:
Got your point. Let me revise and update.